game/scripts/npc/abilities/alchemist_goblins_greed.txt

```diff
@@ -26,17 +26,17 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_gold"                                      "3"
+        "bonus_gold"                                      "3 4 5 6 8 14" //OAA
       }
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_bonus_gold"                                "3 4 5 6 8 11" //OAA
+        "bonus_bonus_gold"                                "3 4 5 6 8 14" //OAA
       }
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_gold_cap"                                  "1000"  //OAA
+        "bonus_gold_cap"                                  "36 42 48 54 66 102" //OAA
```

Comment:
I mean honestly if you're going to change alch so drastically (Which is fine and prob warranted) I would than leave everything unchanged.